apps/gallery/js/frames.js

```diff
@@ -444,7 +444,8 @@ function setupFrameContent(n, frame) {
         frame.displayVideo(videofile, imagefile,
                            fileinfo.metadata.width,
                            fileinfo.metadata.height,
-                           fileinfo.metadata.rotation || 0);
+                           fileinfo.metadata.rotation || 0,
+                           fileinfo.date);
```

Comment:
You shouldn't have to pass date in with an extra parameter. videofile.lastModifiedDate should be the same thing. We get the date from the filesystem, not from file metadata, so just using the File property is probably the easiest way to go here and it avoids modifying the existing API.